multiversx_sdk/core/proto/transaction_pb2.py

```diff
@@ -1,24 +1,183 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: transaction.proto
-"""Generated protocol buffer code."""
+
```

Comment:
Previously, the file was much shorter. 

Maybe we can check previous generation commands vs. current generation command.